HeterogeneousCore/CUDAUtilities/interface/cudaDeviceCount.h

```diff
@@ -0,0 +1,16 @@
+#ifndef HeterogenousCore_CUDAUtilities_cudaDeviceCount_h
+#define HeterogenousCore_CUDAUtilities_cudaDeviceCount_h
+
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+
+#include <cuda_runtime.h>
+
+namespace cudautils {
+  inline int cudaDeviceCount() {
```

Comment:
I guess this should be renamed to `deviceCount()` as well ?

Reply:
Yes, it should.